### ISSUE TITLE
[source-smartsheets]: Add support for reading from Reports

### ISSUE THREAD

**Issue by bffrost** [airbytehq/airbyte#35197]
### Connector Name

source-smartsheets

### Connector Version

1.1.2

### What step the error happened?

Configuring a new connector

## Background
I'm using the Smartsheet Source and have some data that is heavily aggregated from many source Sheets into a Report. I'd like to be able to read from the Report.

## Steps to Reproduce:
- Try configuring a connection, pasting in the ID of a Report rather than a Sheet

## Expected Results:
- I have an option to configure and read from a Report

## Actual Results
- 'ApiError: 404 - Not Found | Check your spreadsheet ID.'

## Extra Information
I am preparing a PR to add this enhancement. I'm aware that the Smartsheet API doesn't allow querying Report rows by modified date, so I've implemented only a Full Refresh sync mode.

### Contribute

- [X] Yes, I want to contribute

**Comment by bffrost** [comment]
@davydov-d Are you open to a community PR from me for adding this feature?

**Comment by marcosmarxm** [comment]
@bffrost do you want to contribute to solve this issue?

**Comment by bffrost** [comment]
@marcosmarxm yes, I do. I have a fix prepared in a fork of the airbyte repo but haven't created a PR yet since the contribution guidelines mentioned PRs not linked to triaged issues would be rejected.

**Comment by octavia-squidington-iii** [comment]
At Airbyte, we seek to be clear about the project priorities and roadmap.  This issue has not had any activity for 180 days, suggesting that it's not as critical as others.  It's possible it has already been fixed. It is being marked as stale and will be closed in 20 days if there is no activity. To keep it open, please comment to let us know why it is important to you and if it is still reproducible on recent versions of Airbyte.

**Comment by bffrost** [comment]
Sorry for the delay, but I've finally submitted a PR with my fix for this

**Comment by octavia-squidington-iii** [comment]
At Airbyte, we seek to be clear about the project priorities and roadmap. This issue has not had any activity for 180 days, suggesting that it's not as critical as others. It's possible it has already been fixed. It is being marked as stale and will be closed in 20 days if there is no activity. To keep it open, please comment to let us know why it is important to you and if it is still reproducible on recent versions of Airbyte.

**Comment by bffrost** [comment]
This was fixed in https://github.com/airbytehq/airbyte/pull/46379
